docs/gallery/scatter3d-with-slider.ipynb

```diff
@@ -100,6 +100,31 @@
     "pp.widgets.Box([fig, slider])"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e6b5f4d3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Use Plopp's widget to slice dimensions\n",
+    "slider = pw.RangeSliceWidget(da, dims=['time'])\n",
+    "slider_node = pp.widget_node(slider)\n",
+    "\n",
+    "# Sum over the selected range of time dimension\n",
+    "slice_node = pw.slice_dims(data_array=da, slices=slider_node)\n",
+    "\n",
+    "sum_slices = pw.operation_dim(data_array=slice_node, operation='sum', dim='time')\n",
```

Comment:
I'm not sure I see the need for having a new function to perform operations.
How about something like
```Py
sum_slices = pp.Node(sc.sum, slice_node, dim='time')
```
?

Reply:
Yeah that makes more sense :)